components/header.tsx

```diff
@@ -12,7 +15,45 @@ export default function Header() {
       />
       <header className="h-header z-10 pt-4 px-4">
         <div className="mx-auto max-w-screen-lg p-4 md:px-8 bg-light rounded-lg">
-          header
+          <ul className="grid grid-cols-2 md:grid-cols-7">
+            <div className="grid grid-cols-2 md:grid-cols-5 md:col-span-4 items-center">
+              <img src="/icons/veahavta-icon.svg" alt="ואהבת לוגו" />
```

Comment:
What will happen to that alt on other languages? We need the alt to come from the CMS as well.